cg.py

```diff
@@ -1,51 +1,68 @@
 """Generate templated Makefile"""
```

Comment:
We should rename this file now that we're taking the project seriously :)

Maybe it's a utility?

Reply:
Let's do it later :D